src/web/components/SlotBodyEnd/BrazeEpic.tsx

```diff
@@ -180,48 +92,64 @@ const BrazeEpic = ({
 			submitComponentEvent({
 				component: {
 					componentType: COMPONENT_TYPE,
-					id: meta.dataFromBraze.ophanComponentId,
+					id: meta.dataFromBraze?.ophanComponentId,
 				},
 				action: 'VIEW',
 			});
 		}
 	}, [hasBeenSeen, meta]);
 
-	if (Epic && meta.dataFromBraze) {
-		// This will come from Braze via the meta from canShow
-		const props = buildEpicProps(meta.dataFromBraze, countryCode);
-
-		if (props) {
-			return (
-				<div ref={setNode} css={wrapperMargins}>
-					<div ref={epicRef}>
-						{/* eslint-disable-next-line react/jsx-props-no-spreading */}
-						<Epic {...props} />
-					</div>
+	if (meta.dataFromBraze) {
+		return (
+			<div ref={setNode} css={wrapperMargins}>
+				<div ref={epicRef}>
+					<BrazeComponent
+						componentName={meta.dataFromBraze.componentName}
+						brazeMessageProps={meta.dataFromBraze}
+						// TODO: Add real implementation for subscribeToNewsletter
+						subscribeToNewsletter={() => {
+							return Math.random() > 0.5
```

Comment:
Do we want to keep this in or just have it return `Promise.resolve()`

Reply:
Ah great question! This is a placeholder until we add the real `subscribeToNewsletter` functionality, which we'll need to do before configuring any of the newsletter epics in prod. I think this was originally just a `Promise.resolve()`, @michaelclapham do you feel strongly about keeping it this way? I think I'm happy to leave as is, or to revert back to simply returning `Promise.resolve()`.

Reply:
Oh yes let's change this back to Promise.resolve()

Reply:
👍  I've gone ahead and reverted to returning `Promise.resolve()`